tools/pubsys/Infra.toml.example

```diff
@@ -0,0 +1,17 @@
+# Path to your root role JSON file.
+root_role_path = "/home/user/root.json"
+
+# You can keep your private key in a file or an SSM parameter; pick one:
+# (Need inline table syntax until this is fixed: https://github.com/alexcrichton/toml-rs/issues/225)
+signing_keys = { default = { file = { path = "/home/user/key.pem" } } }
+# signing_keys = { default = { ssm = { parameter = "/my/parameter" } } }
```

Comment:
Why shouldn't it be under `BUILDSYS_ROOT_DIR`?  What if they're building and using temporary keys for test repos?  I feel like these checks might be assuming too much about workflow..?

Reply:
I was thinking in too general a sense, I suppose; pubsys isn't for general tough repos that you might use outside of an image context, it's meant for images.  Images would have a hard time using a local repo; it'd have to be a qemu mapped path or something.  I'm fine with more restrictions until we have a reason not to.

Reply:
After talking about this offline, for now we agreed to gitignore pem files at the root of the repo, to catch some bad usages, and to add a comment in the example Infra.toml saying not to store keys in the repo.  We agreed not to go further than this for now because we plan to automate role/key creation with a make target, and then gitignore the keys directory, or something similar.